scripts/setup_web1.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# CAUTION: To git clone infogami, environment variables must be set...
+if [[ -z ${GITHUB_TOKEN} ]]; then
+    echo "FATAL: Can not git clone olsystem" ;
+    exit 1 ;
+fi
+
+# apt list --installed
+sudo apt-get update
+sudo apt-get install -y docker.io docker-compose haproxy
+docker --version        # 19.03.8
+docker-compose version  #  1.25.0
+haproxy -v              #  2.0.13-2ubuntu0.1
```

Comment:
Do we need this installed outside the docker?

Reply:
Actually, I just checked ol-web3, and it doesn't look like it uses haproxy at all, so we might be able to remove this entirely.